hamp_radar/serde_pds_files.py

```diff
@@ -0,0 +1,89 @@
+import numpy as np
+from collections.abc import Iterable
+from typing import List
+from pathlib import Path
+
+from geometries import PdsFileGeometry
+from iquick import get_geometry
+
+
+def serialize_pdsfile(filename: Path):
+    from serde.json import to_json
+
+    data = np.memmap(filename, mode="r")
+    mmbgs = list(get_geometry(data))
+    pfg = PdsFileGeometry(filename, mmbgs)
+    return to_json(pfg)
+
+
+def write_serialized_geometry(writefile: Path, geom):
+    import json
+
+    with open(writefile, "w") as json_file:
+        parsed = json.loads(geom)  # for nice formatting
+        json.dump(parsed, json_file, indent=2)
```

Comment:
I'd probably not pretty-print the output. I don't think it will be read by people that often and if you want, you can just run it through `jq`.
If you still like to pretty print it, there should be an option `OPT_INDENT_2` which you can pass through serde's `to_json` to the underlying `orjson` library which should do the trick without deserializing an reserializing the JSON.